backend/src/services/availabilityService.ts

```
@@ -68,51 +68,51 @@
-          }
-        }
+            in: ["SCHEDULED", "CONFIRMED"],
+          },
+        },
       });
 
       const slots: AvailabilitySlot[] = [];
       let currentDate = startOfDay(startDate);
 
       while (currentDate <= endOfDay(endDate)) {
         const dayOfWeek = this.getDayOfWeek(currentDate);
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)